server/api/initial_data.py

```diff
@@ -867,6 +871,29 @@ def _migrate_model_monitoring_jobs(db, db_session):
     )
 
 
+def _perform_version_7_data_migrations(
+    db: server.api.db.sqldb.db.SQLDB, db_session: sqlalchemy.orm.Session
+):
+    _migrate_project_summaries(db, db_session)
+
+
+def _migrate_project_summaries(db, db_session):
```

Comment:
We're not actually migrating anything
```suggestion
def _create_project_summaries(db, db_session):
```